avocado/plugins/run.py

```diff
@@ -224,6 +225,8 @@ def run(self, args):
                 job_run = job_instance.run()
             finally:
                 # Run JobPost plugins
+                if args.output_check_record:
```

Comment:
This is fine... but, to be even more in line with Avocado plugin mechanism, it'd make sense to make this a plugin itself.  The plugin can check if the option was given and then merge the files.  With that, this line would not be necessary.

Reply:
Ok, I will create a plugin for this.